features/insured/step_definitions/individual_curam_document_steps.rb

```diff
@@ -4,23 +4,23 @@
 end
 
 Then(/^a button will be visible to the user labeled MEDICAID & TAX CREDITS$/) do
-  expect(page).to have_content('Medicaid & Tax Credits')
+  expect(page).to have_content(AdminHomepage.medicaid_button_text)
```

Comment:
let's remove all `have_content` in favor of attribute selectors like [data-cuke=""]